src/client.js

```diff
@@ -47,7 +47,16 @@ function generateIrx() {
 function login() {
     let key = utils.sanitizeString(process.env.INPUT_ASOC_KEY);
     let secret = utils.sanitizeString(process.env.INPUT_ASOC_SECRET);
-    return executeCommand(`api_login -u ${key} -P ${secret} `);
+    if (process.env.INPUT_SERVICE_URL != null) {
```

Comment:
I suggest using `if(process.env.INPUT_SERVICE_URL)` without the `!= null` check since the latter may behave unexpectedly.
Also, since there are 3 possibly different commands that only differ by the options at the end, it would be cleaner to declare a variable that holds the options (e.g. `let options = ''`) and append to those options as needed. That would make it easier in the future if we have more options that need to be added to the command and it means there's only 1 instance of the command if it ever needs to be modified in the future.